pkg/nsm/apis/nseconnect/nseconnect.proto

```diff
@@ -18,8 +18,24 @@ import "github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh/netmesh.proto"
         string linux_namespace = 3;
     }
 
+// EndpointConnectionInterface is sent by a NSM to NSE to inform NSE about
+// the interface name which was created by the dataplane controller.
+    message EndpointConnectionInterface {
+        string request_id = 1;
+        string interface_name = 2;
+        string network_service_name = 3;
+    }
+    
+// EndpointConnectionInterfaceReply is sent back by NSE to NSM to confirm the presence 
+// of the programmed interface.
+    message EndpointConnectionInterfaceReply {
+        string request_id = 1;
+        bool interface_found = 2;
+    }
+
     service EndpointConnection {
         rpc RequestEndpointConnection (EndpointConnectionRequest) returns (EndpointConnectionReply);
+        rpc SendEndpointConnectionInterface (EndpointConnectionInterface ) returns (EndpointConnectionInterfaceReply);
```

Comment:
Thinking a bit about this... is there any reason we can't have figured out what we need to know to send it with the EndpointConnectionRequest ?

If at all possible, I'd love to keep this to a single request reply :)

Reply:
I do not see why second message is a problem, but guessing might lead to crashes, what could be more reliable than the interface name reported back by the dataplane to nsm? I think reliability is the priority.